src/legacy/storage.ts

```diff
@@ -79,6 +79,15 @@ export const keys = async (path: string, includeSubdirs?: boolean): Promise<Arra
   }
 }
 
+const storage = {
```

Comment:
maybe export the typeof

Reply:
for what?

Reply:
many places are extracting the type `typeof storage` DRY principle.

Reply:
this is a pretty big change, we can do that later

Reply:
g2 agree